test/archetype.test.ts

```diff
@@ -9,6 +9,7 @@ import {
   Factory__factory,
 } from "../typechain";
 import Invitelist from "../lib/invitelist";
+import Blacklist from "../lib/blacklist";
```

Comment:
not sure this new file is necessary. blacklist.ts is an exact duplicate of invitelist.ts, so you can probably just reuse Invitelist class on line 2056.